test/controllers/GuidanceControllerSpec.scala

```diff
@@ -73,17 +74,17 @@ class GuidanceControllerSpec extends BaseSpec with GuiceOneAppPerSuite {
 
     "return an OK response" in new QuestionTest {
       MockGuidanceService
-        .getPage(path, processId, None)
-        .returns(Future.successful(Some(expectedPage)))
+        .getPageContext(path, processId, None)
+        .returns(Future.successful(Some(PageContext(expectedPage, "/"))))
 
       val result: Future[Result] = target.getPage(relativePath)(fakeRequest)
```

Comment:
IntelliJ reports a mismatch between the expected and actual return types. The required response is Future[Result], but the actual return type is Accumulator[ByteString, Result]. Can you explain this?